ledger/eval.go

```diff
@@ -274,6 +274,13 @@ func (eval *BlockEvaluator) Round() basics.Round {
 	return eval.block.Round()
 }
 
+// ResetTxnBytes resets the number of bytes tracked by the BlockEvaluator to
+// zero.  This is a specialized operation used by the transaction pool to
+// simulate the effect of putting pending transactions in multiple blocks.
+func (eval *BlockEvaluator) ResetTxnBytes(validateTxnBytes bool) {
+	eval.totalTxBytes = 0
```

Comment:
should this increment eval.Round somehow?

Reply:
I considered that (that would be a better API for the transaction pool, to be able to simulate several future blocks, instead of the `pendingBlockEvaluatorRound` kludge that I currently have there).  But it turns out that changing `eval.Round()` leads to other complications in the block evaluator that didn't seem worth it; the block evaluator more-or-less assumes it executes all for a single round right now.